binary/util.py

```diff
@@ -300,6 +301,12 @@ def encode_string_frame(client_message, value=None):
     def encode_data_frame(client_message):
         client_message.add_frame(Frame(reference_objects.DATA))
 
+    @staticmethod
+    def encode_json(client_message):
+        client_message.add_frame(BEGIN_FRAME)
+        client_message.add_frame(Frame("{'value': ''}".encode('utf-8')))
+        client_message.add_frame(END_FRAME)
+
```

Comment:
I guess we won't need those anymore

Reply:
I believe we need this for proper usage of ```HazelcastJsonValue```. That type was introduced in ```2.4```, but nobody uses it. When I tried to use it the generated tests failed.